pkgs/top-level/perl-packages.nix

```
@@ -12003,6 +12003,14 @@ let
       url = "mirror://cpan/authors/id/P/PE/PEVANS/IO-Async-SSL-0.23.tar.gz";
       hash = "sha256-0vyuFuJ+F6yjkDpK1aK/L7wmjQZRzn8ARabQVG9YTy4=";
     };
+    patches = [
+      (fetchpatch {
+        # Fixes test compatibility with OpenSSL 3
+        url = "https://sources.debian.org/data/main/libi/libio-async-ssl-perl/0.23-1/debian/patches/upgrade-error-match.patch";
```

```suggestion
      # Fixes test compatibility with OpenSSL 3
      (fetchpatch {
        url = "https://sources.debian.org/data/main/libi/libio-async-ssl-perl/0.23-1/debian/patches/upgrade-error-match.patch";
```

thats only really important for CVE comments I think

---

The purpose of the patch is not clear from the `name`, i think it's useful to have this as a comment. Also, keeping the comment over the specific `fetchpatch` makes more sense to me and is consistent with perlPackages in general, imho.

---

I meant that I think for the CVE detection to the work the comment must be before the fetchpatch, not somewhere in the middle.